lib/crystalball/simple_predictor.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'crystalball/predictor'
+require 'crystalball/predictor/modified_execution_paths'
+require 'crystalball/predictor/modified_specs'
+
+module Crystalball
+  # Class to predict test failures with given execution map and sources diff
+  class SimplePredictor < Predictor
```

Comment:
We can remove https://github.com/toptal/crystalball/blob/master/features/fixtures/simple_app/standard_predictor.rb in favor of this one